2021-02-17 10:49:12.142792: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-17 10:49:18.129773: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcuda.so.1
2021-02-17 10:49:19.166063: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:19.177583: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:04:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.545GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-02-17 10:49:19.177632: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-17 10:49:19.219261: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-17 10:49:19.264392: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2021-02-17 10:49:19.282841: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2021-02-17 10:49:19.348810: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2021-02-17 10:49:19.360582: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2021-02-17 10:49:19.405105: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-17 10:49:19.405369: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:19.406821: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:19.408067: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
Using TensorFlow version:  2.3.0 , GPU: 1
Using Keras version:  2.4.0
2021-02-17 10:49:19.629075: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 2499965000 Hz
2021-02-17 10:49:19.631670: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4dede00 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-02-17 10:49:19.631753: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-02-17 10:49:19.761051: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:19.762227: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x4e5a240 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2021-02-17 10:49:19.762330: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce RTX 2080 Ti, Compute Capability 7.5
2021-02-17 10:49:19.762872: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:19.764748: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1716] Found device 0 with properties: 
pciBusID: 0000:04:00.0 name: GeForce RTX 2080 Ti computeCapability: 7.5
coreClock: 1.545GHz coreCount: 68 deviceMemorySize: 10.76GiB deviceMemoryBandwidth: 573.69GiB/s
2021-02-17 10:49:19.764822: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-17 10:49:19.764886: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-17 10:49:19.764955: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcufft.so.10
2021-02-17 10:49:19.765003: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcurand.so.10
2021-02-17 10:49:19.765047: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusolver.so.10
2021-02-17 10:49:19.765091: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcusparse.so.10
2021-02-17 10:49:19.765135: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-17 10:49:19.765284: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:19.766834: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:19.768059: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1858] Adding visible gpu devices: 0
2021-02-17 10:49:19.768123: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.10.1
2021-02-17 10:49:21.201418: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1257] Device interconnect StreamExecutor with strength 1 edge matrix:
2021-02-17 10:49:21.201520: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1263]      0 
2021-02-17 10:49:21.201542: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1276] 0:   N 
2021-02-17 10:49:21.202009: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:21.203579: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:982] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2021-02-17 10:49:21.205018: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1402] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10068 MB memory) -> physical GPU (device: 0, name: GeForce RTX 2080 Ti, pci bus id: 0000:04:00.0, compute capability: 7.5)
The inputs are normalized to have a unit Gaussian distribution
Model: "CNN-POD"
_________________________________________________________________
Layer (type)                 Output Shape              Param #   
=================================================================
high-res-input (InputLayer)  [(None, 3, 48, 48)]       0         
_________________________________________________________________
predic_01 (Conv2D)           (None, 128, 48, 48)       9728      
_________________________________________________________________
predic_02 (BatchNormalizatio (None, 128, 48, 48)       512       
_________________________________________________________________
predic_03 (MaxPooling2D)     (None, 128, 24, 24)       0         
_________________________________________________________________
predic_04 (Conv2D)           (None, 256, 24, 24)       295168    
_________________________________________________________________
predic_05 (BatchNormalizatio (None, 256, 24, 24)       1024      
_________________________________________________________________
predic_06 (MaxPooling2D)     (None, 256, 12, 12)       0         
_________________________________________________________________
predic_07 (Conv2D)           (None, 256, 12, 12)       590080    
_________________________________________________________________
predic_08 (BatchNormalizatio (None, 256, 12, 12)       1024      
_________________________________________________________________
predic_10 (Conv2D)           (None, 512, 12, 12)       1180160   
_________________________________________________________________
predic_11 (BatchNormalizatio (None, 512, 12, 12)       2048      
_________________________________________________________________
predic_13 (Conv2D)           (None, 512, 12, 12)       2359808   
_________________________________________________________________
predic_14 (BatchNormalizatio (None, 512, 12, 12)       2048      
_________________________________________________________________
predic_15 (Conv2D)           (None, 64, 12, 12)        294976    
=================================================================
Total params: 4,736,576
Trainable params: 4,733,248
Non-trainable params: 3,328
_________________________________________________________________
None
2021-02-17 10:49:43.177117: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 1865 of 4200
2021-02-17 10:49:53.150633: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 3922 of 4200
2021-02-17 10:49:54.584775: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
2021-02-17 10:49:56.289602: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudnn.so.7
2021-02-17 10:49:59.749867: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcublas.so.10
2021-02-17 10:53:32.854859: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 1969 of 4200
2021-02-17 10:53:42.838536: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 4012 of 4200
2021-02-17 10:53:43.718159: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
Epoch 0001/0020, loss: 6.718275547027588, val_loss: 2.2298641204833984, elapsed time from start: 284.9976758956909
2021-02-17 10:57:10.085415: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:172] Filling up shuffle buffer (this may take a while): 3297 of 4200
2021-02-17 10:57:13.694537: I tensorflow/core/kernels/data/shuffle_dataset_op.cc:221] Shuffle buffer filled.
Epoch 0002/0020, loss: 0.9794734120368958, val_loss: 0.32364749908447266, elapsed time from start: 495.1535613536835
Epoch 0003/0020, loss: 0.15117859840393066, val_loss: 0.060556381940841675, elapsed time from start: 596.1814525127411
Epoch 0004/0020, loss: 0.03574017062783241, val_loss: 0.02303500473499298, elapsed time from start: 694.072503566742
Epoch 0005/0020, loss: 0.017568763345479965, val_loss: 0.01513885147869587, elapsed time from start: 794.2535643577576
Epoch 0006/0020, loss: 0.01421680860221386, val_loss: 0.013996932655572891, elapsed time from start: 898.7169167995453
Epoch 0007/0020, loss: 0.013333193026483059, val_loss: 0.01336208637803793, elapsed time from start: 1001.4087600708008
Epoch 0008/0020, loss: 0.012979735620319843, val_loss: 0.013117465190589428, elapsed time from start: 1101.8174092769623
Epoch 0009/0020, loss: 0.012769926339387894, val_loss: 0.01299519743770361, elapsed time from start: 1201.7780768871307
Epoch 0010/0020, loss: 0.012612752616405487, val_loss: 0.012826229445636272, elapsed time from start: 1301.8817467689514
Epoch 0011/0020, loss: 0.012482677586376667, val_loss: 0.012778664007782936, elapsed time from start: 1401.6676404476166
Epoch 0012/0020, loss: 0.012322974391281605, val_loss: 0.012592427432537079, elapsed time from start: 1502.1799097061157
Epoch 0013/0020, loss: 0.012187917716801167, val_loss: 0.012428311631083488, elapsed time from start: 1603.8649394512177
Epoch 0014/0020, loss: 0.012075086124241352, val_loss: 0.01228394079953432, elapsed time from start: 1704.329912185669
Epoch 0015/0020, loss: 0.011977951973676682, val_loss: 0.012158514931797981, elapsed time from start: 1804.7804746627808
Epoch 0016/0020, loss: 0.011891169473528862, val_loss: 0.012147392146289349, elapsed time from start: 1907.5333578586578
Epoch 0017/0020, loss: 0.011818374507129192, val_loss: 0.011987755075097084, elapsed time from start: 2007.2161271572113
Epoch 0018/0020, loss: 0.011753718368709087, val_loss: 0.01201762817800045, elapsed time from start: 2109.9095799922943
Epoch 0019/0020, loss: 0.011693617329001427, val_loss: 0.011888994835317135, elapsed time from start: 2213.2625770568848
Epoch 0020/0020, loss: 0.011640544980764389, val_loss: 0.011853781528770924, elapsed time from start: 2313.3812210559845
WARNING:tensorflow:From /home/alejandro/projects/urban-flows/.env/lib/python3.7/site-packages/tensorflow/python/training/tracking/tracking.py:111: Model.state_updates (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
2021-02-17 11:28:09.025527: W tensorflow/python/util/util.cc:348] Sets are not currently considered sequences, but this may change in the future, so consider avoiding using them.
WARNING:tensorflow:From /home/alejandro/projects/urban-flows/.env/lib/python3.7/site-packages/tensorflow/python/training/tracking/tracking.py:111: Layer.updates (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.
Instructions for updating:
This property should not be used in TensorFlow 2.0, as updates are applied automatically.
